sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -422,7 +417,7 @@ def test_series_tests(self):
             # Throws NotImplementedError when modifying df
             'pandas.core.series.Series.transform': ['df'],
             'pandas.core.series.Series.autocorr': ['*'],
-            'pandas.core.series.Series.combine': ['*'],
+            #'pandas.core.series.Series.combine': ['*'],
```

Comment:
Please go ahead and remove this

Reply:
Thanks, @TheNeuralBit - Sorry for that, I fix it. And I did what you said, but the test still fail in the workflow, I don't know why, I'm running Pytest locally and all tests pass. Could you help me seen what I'm missing?